api/3.12.3/docs/index.html

```diff
@@ -93,7 +93,7 @@ <h3 id="hazelcast">Hazelcast</h3>
 				<pre><code class="lang-bash">docker run -p 5701:5701 hazelcast/hazelcast:3.12.6
```

Comment:
In this file you need to fix other links. For example: https://github.com/hazelcast/hazelcast-nodejs-client/blob/be17452c2edaef1f7a49b9d379f82c06c1df78c5/api/3.12.3/docs/index.html#L198


Can you check all files for relative links?

Reply:
You need to check all files because we might have changed the README in master in any version.

Reply:
Do I have to check every href? Sorry yo be that late. Since not in Jira I missed this PR

Reply:
I think that's the case. You can exclude the links that start with `http`. However, you need to check all links in all readmes.